.github/workflows/coverage.yml

```diff
@@ -14,33 +14,21 @@ jobs:
         os: [ ubuntu-latest ]
 
     steps:
-      - name: Git Checkout
-        uses: actions/checkout@v3
+      - name: Git checkout
+        uses: actions/checkout@v4
         with:
           submodules: recursive
-      - name: Rust Toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
```

Comment:
🗺️ The GH Actions owned by the `actions-rs` organization are now [archived and unmaintained](https://github.com/actions-rs). `dtolnay/rust-toolchain` is its successor.